student.rb

```diff
@@ -1,13 +1,20 @@
 require_relative 'person'
+require_relative 'classroom'
 
 class Student < Person
   attr_accessor :name, :age, :classroom
+  attr_reader :label
 
   def initialize(name, age, classroom, parent_permission: true)
     super(name, age, parent_permission: parent_permission)
     @classroom = classroom
   end
 
+  def label=(label)
+    @label = label
+    label.students.push(self) unless label.students.include?(self)
+  end
```

Comment:
Here, remember that the relationship should be between students and classrooms and not labels. So kindly update it accordingly. :+1: